.github/workflows/labeler.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@main
+    - uses: actions/labeler@v3.0.2
```

Comment:
Judging by https://github.com/evverx/systemd/pull/10/files, Dependabot supports pinning GHActions to specific SHAs. I'll try to switch to SHAs instead of tags